index.html

```diff
@@ -47,6 +47,148 @@ <h2 class="secondary-text">I'am a Software Developer</h2>
     </div>
   </section>
   </div>
-</body>
 
+  <div class="works">
+    <div class="title">
+      <a id="recent" >My Recent Works</a>
+      <div class="line"><img src="Images\line.png" alt="line.png"></div>
+    </div>
+    <div class="frame">
+      <div class="featured">
+        <div class="img">
+        </div>
+        <div class="block">
+            <div class="multi"> Multi-Post Stories </div>
+            <div class="sup">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+        </div>
+        <ul class="textblock-tags">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <div class="proj"><a href="#" id="project">
+          <img src="Images\Seeproj.png" alt="proj.png"></a></div>
+      </div>
+      <div class="frame43">
+        <div class="content-defaults content1">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content2">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
```

Comment:
Please fix the colors for this part to match the figma design. Currently your content is not displaying

<img width="358" alt="Screenshot 2022-10-29 at 18 26 36" src="https://user-images.githubusercontent.com/48546992/198839938-46523dd1-7ef3-4669-af74-93cb87bde52f.png">

Reply:
Again on my computer, everything is working fine. see this image 
![Untitled1](https://user-images.githubusercontent.com/114238556/198841483-0e80a7a3-36b6-4322-9f80-76a2c69fe8b0.png)

Reply:
@Nyame-Wolf Please help me again. Thanks You

Reply:
Don't worry. I understand what happened here too. I can see it.